src/olympia/accounts/views.py

```diff
@@ -303,6 +303,8 @@ def inner(self, request):
                     request, ERROR_NO_PROFILE, next_path=next_path, format=format
                 )
             else:
+                # The following log statement is used by foxsec-pipeline.
+                log.info('Logging in FxA user %s', identity['email'])
```

Comment:
Note: this is a reworded version of what was there before, with only the relevant info needed by foxsec-pipeline.

Reply:
https://github.com/mozilla-services/foxsec-pipeline/issues/510 goes with that change

Reply:
oops sorry about that, good call on adding comments for the future!